File: forense/web/i18n-es-cl.js
/** Textos Forense — español de Chile (es-CL). */

export const STATUS = {
  queued: "en cola",
  processing: "procesando",
  done: "completado",
  error: "error",
};

export const KIND = {
  person: "persona",
  machinery: "maquinaria",
  other: "otro",
};

export const EVENT_TYPE = {
  action: "acción",
  epp_non_compliant: "incumplimiento EPP",
  zone: "zona restringida",
  speed_violation: "exceso de velocidad",
  proximity: "proximidad crítica",
  knowledge_match: "coincidencia biblioteca",
  knowledge_conjecture: "conjetura biblioteca",
  collision: "colisión",
  fall_risk: "riesgo de caída",
  unsafe_act: "acto inseguro",
};

export const SOURCE = {
  user: "usuario",
  live: "video en vivo",
  seed: "plantilla",
  osha: "OSHA",
};

export function statusLabel(code) {
  return STATUS[code] || code || "—";
}

export function kindLabel(code) {
  return KIND[code] || code || "—";
}

export function eventTypeLabel(code) {
  return EVENT_TYPE[code] || (code || "—").replace(/_/g, " ");
}

export function sourceLabel(code) {
  return SOURCE[code] || code || "";
}

File: forense/app/i18n_es_cl.py
"""Etiquetas en español de Chile para informes y API Forense."""

from __future__ import annotations

STATUS_LABELS: dict[str, str] = {
    "queued": "en cola",
    "processing": "procesando",
    "done": "completado",
    "error": "error",
}

KIND_LABELS: dict[str, str] = {
    "person": "persona",
    "machinery": "maquinaria",
    "other": "otro",
}

EVENT_TYPE_LABELS: dict[str, str] = {
    "action": "acción",
    "epp_non_compliant": "incumplimiento EPP",
    "zone": "zona restringida",
    "speed_violation": "exceso de velocidad",
    "proximity": "proximidad crítica",
    "knowledge_match": "coincidencia biblioteca",
    "knowledge_conjecture": "conjetura biblioteca",
    "collision": "colisión",
    "fall_risk": "riesgo de caída",
    "unsafe_act": "acto inseguro",
    "fire": "incendio / llamas",
    "smoke": "humo",
    "epp_reflective": "sin ropa reflectante",
    "emergency_response": "respuesta de emergencia",
    "collision": "colisión / golpe",
}

SEVERITY_LABELS: dict[str, str] = {
    "critical": "crítica",
    "high": "alta",
    "medium": "media",
    "low": "baja",
}


def label_event_type(code: str) -> str:
    return EVENT_TYPE_LABELS.get(code, (code or "—").replace("_", " "))


def label_kind(code: str) -> str:
    return KIND_LABELS.get(code, code or "—")


def label_severity(code: str) -> str:
    return SEVERITY_LABELS.get(code, code or "—")
